### DISCUSSION TITLE
Download Shared Projects

### DISCUSSION THREAD

**Discussion by FazahChamp** [ScratchAddons/ScratchAddons#1437] [Ideas]
Please Add "Download Shared Projects" Button On My Stuff So It Will Be Easy To Download Our Shared Projects As A ZIP File.

PLEASE ADD THIS!
PLEASE!
PLEASE!
PLEASE!

Thanks @Fazah_Champ

**Comment by Marc92020** [comment]
Chill. They will talk about it first than they might add it. But it will be for 1.9.0 if they do add it.

**Reply by FazahChamp** [reply]
Ok I am Waiting For It

**Comment by j5155** [comment]
If you program it they'll probably add it

**Reply by FazahChamp** [reply]
Should I Write Codes???

**Comment by cobaltt7** [comment]
> But it will be for 1.9.0 if they do add it.

Unless it’s not made in time

**Comment by ghost** [comment]
Yes, yes, yes. This will be SO useful, especially because I recently switched from my old account on which I have 100+ projects.

**Comment by cobaltt7** [comment]
Maybe a bulk upload as well?
________________________________
